packages/table/src/cell/formats/jsonFormat.tsx

```diff
@@ -29,9 +28,9 @@ export interface IJSONFormatProps extends ITruncatedFormatProps {
     stringify?: (obj: any) => string;
 }
 
-@PureRender
-export class JSONFormat extends React.Component<IJSONFormatProps, {}> {
+export class JSONFormat extends React.PureComponent<IJSONFormatProps, {}> {
```

Comment:
👎 you cannot use `PureComponent` because our React peer dependency is `"react": "^15.0.1 || ^0.14",`: we support versions of React that do not provide `PureComponent`.

please revert to `@PureRender`.